cardano-ledger-test/src/Test/Cardano/Ledger/Examples/TwoPhaseValidation.hs

```diff
@@ -1424,8 +1423,8 @@ testUTXOW tx predicateFailure@(Left _) = do
             (TRC (utxoEnv (Alonzo Mock), (initialUtxoSt $ Alonzo Mock), tx))
   st @?= predicateFailure
 
-trustMe :: Bool -> Core.Tx A -> ValidatedTx A
-trustMe v (Core.Tx b w a) = ValidatedTx b w (IsValidating v) a
+trustMe :: Bool -> ValidatedTx A -> ValidatedTx A
+trustMe iv' (ValidatedTx b w _ m) = ValidatedTx b w (IsValidating iv') m
```

Comment:
I think you do not need this function anymore, the tests can just stamp the correct `IsValidating` field directly.